### PR TITLE
Feature/mission-controls-button-styles

### PR DESCRIPTION
**Pull request by Julianisrel**
https://trello.com/c/obzk8UyU/650-fix-styling-on-modal

1) match button styling to rest of web app
2) increase font size of button, make more readable
3) center button
4) increase size of "x", move to the top, right of the modal
5) make modal responsive
6) make the button match the theme of the other buttons in this app for design consistency

### CODE REVIEW

File: src/components/pages/HeadsUpDisplay/Hud.js [hunk 1]
```diff
@@ -32,7 +32,13 @@ export default function (props) {
               ${
                 completedActivity.length - 1 >= activities.indexOf(a) &&
                 'completedActivity'
-              }`}
+              }
```
Comment (just-kato) [inline src/components/pages/HeadsUpDisplay/Hud.js]: Great job with making the styling look consistent throughout the application

File: src/styles/less/Hud.less [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
 :root{
-	--green: #C6FD7E;
+	--green: #c6fd7e;
 	--grey: #c7c7c7;
 	--white: white;
 	--bar-border-radius: 200px;
-	--min-height: .75rem;
-	--font-size: 1.5rem;
+	--min-height: 0.75rem;
+	--font-size: 1.7rem;
```
Comment (just-kato) [inline src/styles/less/Hud.less:10]: Nice work

File: src/styles/less/Hud.less [hunk 3]
```diff
@@ -32,7 +33,7 @@
     display: flex;
     flex-flow: row, wrap;
     color: black;
-    background-color: white;
+    background-color: var(--green);
```
Comment (just-kato) [inline src/styles/less/Hud.less]: Nice job withe the background color change

File: src/styles/less/Hud.less [hunk 4]
```diff
@@ -47,11 +48,11 @@
...
     color: black;
-    //background-color: white;
-    background-image: linear-gradient(to right,  var(--green) 0%, white  5%;);
+    // background-color: white;
+    background-image: var(--green);
     font-size: var(--font-size);
-	font-weight: 500;
-	font-family: 'Bangers', cursive;
+	  font-weight: 500;
+	  font-family: 'Bangers', cursive;
```
Comment (just-kato) [inline src/styles/less/Hud.less]: Nice work

File: src/styles/less/Hud.less [hunk 5]
```diff
@@ -247,4 +253,30 @@
 // State not done for this PT21
 .completedDay {
     background-color: var(--green);
-}
\ No newline at end of file
+}
+
+
```
Comment (just-kato) [inline src/styles/less/Hud.less]: This is huge. Making the app responsive. Nice work!

File: src/styles/less/Hud.less [hunk 6]
```diff
@@ -127,11 +129,15 @@
   display: none;
 }
 
+.restActive {
+	padding: 8px;
```
Comment (just-kato) [inline src/styles/less/Hud.less:157]: Nice job

**Review by just-kato — DISMISSED** [review]

**Comment by tinomen** [comment]
You need to rollback (revert) `package-lock.json` before we can merge into main.

**Comment by ethankarp** [comment]
Hey @Julianisrel, If you run `git checkout origin/main package-lock.json` the file should get restored. I think that command will also automatically stage the file, so then just commit and push and you should be good.

File: src/styles/less/Hud.less (hunk 5 above)
Comment (Yonathan-Admasu728) [reply src/styles/less/Hud.less]: Nice!!!!

File: src/styles/less/Hud.less (hunk 4 above)
Comment (Yonathan-Admasu728) [reply src/styles/less/Hud.less]: clean

**Review by Yonathan-Admasu728 — DISMISSED** [review]

**Review by MichaelMcFaddenjr — DISMISSED** [review]
nice clean code!

**Review by ethankarp — DISMISSED** [review]

**Review by rhiannonstanford — DISMISSED** [review]
Nice clean up.  Thank you!

File: src/styles/less/Hud.less [hunk 7]
```diff
@@ -8,14 +8,14 @@
...
 	--font-size: 1.7rem;
-	letter-spacing: .2rem;
 }
 
 .HudContainer {
     width: 100%;
     background-color: #757575;
     display: flex;
     flex-direction: column;
+    letter-spacing: .2rem;
```
Comment (ghost) [inline src/styles/less/Hud.less:18]: Good work figuring out this would be better in the container than the root.

**Review by ghost — DISMISSED** [review]

**Review by ethankarp — APPROVED** [review]

**Review by rhiannonstanford — APPROVED** [review]
Good job resolving the conflict.

**Pull request merged**